build/build-image/Dockerfile

```diff
@@ -20,8 +20,8 @@ RUN curl -sSL -o /usr/local/bin/dep https://github.com/golang/dep/releases/downl
     chmod +x /usr/local/bin/dep
 
 # Install etcd
-RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.10/etcd-v3.1.10-linux-amd64.tar.gz \
-    | tar -vxz -C /usr/local/bin --strip=1 etcd-v3.1.10-linux-amd64/etcd
+RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.2.24/etcd-v3.2.24-linux-amd64.tar.gz \
```

Comment:
Nice! Consistent with the one used in Kubernetes
https://github.com/kubernetes/kubernetes/blob/release-1.12/build/root/WORKSPACE#L24

Reply:
Possibly also define as a variable instead of duplicating 3 times? Could be done as a follow-up.